### DISCUSSION TITLE
"Sync for Receivables" type product

### DISCUSSION THREAD

**Discussion by rfazzo** [codatio/codat-docs#974] [Q&A] [answered]
Is there any plans to depreciate the accounting APIs that codat current have? I see those API's are no longer accessible to new customers as of Sep 2023. 

If so, is there a product in the pipeline along the line of "Sync for Receivables" where we can continue to sync invoices from our product to our customers accounting products?

TIA

**Comment by dcoplowe** [comment, marked as answer]
Hi @rfazzo,

I apologise for any inconvenience caused by the delay.

For assistance regarding future support for clients using the Accounting API and how we can better support you, please feel free to reach out to your dedicated Codat contact. Alternatively, you can email our support team at support@codat.io. They are well-equipped to address your queries and provide the necessary support.

Many thanks,

David
